### PR TITLE
Use travis for windows tests

### PR DESCRIPTION
**Pull request by pavanteja-potnuru**
browser tests now use puppeteer.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -143,7 +143,7 @@
     "no-mixed-requires": "off",
     "no-new-require": "off",
     "no-path-concat": "error",
-    "no-process-env": "error",
```
Comment (kunagpal) [inline .eslintrc]: No.

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +1,13 @@
 dist: trusty
```
Comment (kunagpal) [inline .travis.yml]: Add the `os` parameter as well.

File: test/karma.conf.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 // Karma configuration
 // Generated on Mon Nov 09 2015 18:53:12 GMT+0530 (IST)
+process.env.CHROME_BIN = require('puppeteer').executablePath();
```
Comment (kunagpal) [inline test/karma.conf.js]: Skip the ESLint rule for this line, instead of disabling it altogether.

File: .travis.yml [hunk 4]
```diff
@@ -1,18 +1,13 @@
...
 language: node_js
 node_js:
-- '4'
 - '6'
 - '8'
+- '10'
 addons:
   apt:
```
Comment (codenirvana) [inline .travis.yml]:
```suggestion
  chrome: stable
```

**Pull request merged**